MISSION.md

```diff
@@ -0,0 +1,46 @@
+# Xdev Mission & Vision
+
+This document is meant to outline the current vision of the future of
+scientific data analysis in the geosciences, and it is meant to lay out
+the goals and necessary plans to achieve this vision.  This vision
+should be seen as a *forecast* from the Team's perspective, so everyone's
+input is welcome.  As the Engineering Lead for the Xdev Team, we do not
```

Comment:
With the switch from "I" to "we", I think you (we?) should drop the "As the Engineering Lead for the Xdev Team" clause in this sentence.

Reply:
Agreed.  Thanks for the catch.